File: extensions/instagram/src/main/java/app/revanced/extension/instagram/profile/CopyBioTextPatch.java
package app.revanced.extension.instagram.profile;

import android.widget.TextView;

@SuppressWarnings("unused")
public class CopyBioTextPatch {

    /**
     * Injection point.
     * Makes the bio TextView selectable and copyable.
     */
    public static void makeBioTextSelectable(TextView bioTextView) {
        if (bioTextView != null) {
            bioTextView.setTextIsSelectable(true);
        }
    }
}

File: patches/src/main/kotlin/app/revanced/patches/instagram/profile/copybio/CopyBioTextPatch.kt
package app.revanced.patches.instagram.profile.copybio

import app.revanced.patcher.patch.bytecodePatch
import app.revanced.util.returnEarly

@Suppress("unused")
val copyBioTextPatch = bytecodePatch(
    name = "Copy bio text",
    description = "Makes user bio text selectable and copyable on profile pages.",
    use = false,
) {
    compatibleWith("com.instagram.android")

    apply {
        profileBioMethod.returnEarly()
    }
}
